from db_config import conectar

try:
    conn = conectar()
    print("Conex√£o bem-sucedida!")
    conn.close()
except Exception as e:
    print(f"Erro ao conectar: {e}")
